AdvLoggerPkg/Library/AdvancedLoggerLib/MmCoreArm/AdvancedLoggerLib.c

```diff
@@ -15,63 +15,15 @@
 #include <Library/BaseMemoryLib.h>
 #include <Library/DebugLib.h>
 #include <Library/SynchronizationLib.h>
-#include <Library/MmuLib.h>
 
 #include "../AdvancedLoggerCommon.h"
 
 #define ADV_LOGGER_MIN_SIZE  (65536)

```

Comment:
I don't see `ADV_LOGGER_MIN_SIZE` used. Is it needed?

Reply:
Will remove. Can't think of a reason it would be needed.